### PR TITLE
Docs: keyboard shortcuts.

### PR DESCRIPTION
**Pull request by godai78**
Making guide intro human-friendly.

Adding related productivity features.

### CODE REVIEW

File: docs/features/keyboard-support.md [hunk 1]
```diff
@@ -5,8 +5,12 @@ classes: keyboard-shortcuts
 
 # Keyboard support
 
+CKEditor 5 offers plenty of features that aid editing process and help streamline and speed up writing. Using keyboard shortcuts is one of the simplest yet most efficient and popular ways to improve interaction with the software.
```
Comment (AnnaTomanek) [inline docs/features/keyboard-support.md:8]:
```suggestion
CKEditor 5 offers plenty of features to help you create your content in the most productive way. Using keyboard shortcuts is one of the simplest yet most efficient and popular ways to improve interaction with the software.
```

File: docs/features/keyboard-support.md [hunk 2]
```diff
@@ -5,8 +5,12 @@ classes: keyboard-shortcuts
...
+
 Below is a list of the most important keystrokes supported by CKEditor 5 and its features:
```
Comment (AnnaTomanek) [inline docs/features/keyboard-support.md:10]:
```suggestion
Below is a list of the most important keystrokes supported by CKEditor 5 and its features.
```

File: docs/features/keyboard-support.md [hunk 3]
```diff
@@ -156,3 +165,13 @@ Use the following keystrokes for more efficient navigation in the CKEditor 5 use
 	width: 30%;
 }
 </style>
+
+
+## Related productivity features
+
+There are more features that aid the efficiency and improve productivity:
```
Comment (AnnaTomanek) [inline docs/features/keyboard-support.md:172]:
```suggestion
In addition to using keyboard shortcuts, you may want to check the following productivity features:
```

**Comment by AnnaTomanek** [comment]
Paste plain text keyboard shortcut is **missing** on the list.

Also, maybe we can mention two-step caret movement that allows you to [get out of links](https://ckeditor.com/docs/ckeditor5/latest/features/link.html#typing-around-links) and formats?

**Comment by godai78** [comment]
The two-step caret is kinda tricky.

**Review by AnnaTomanek — APPROVED** [review]

**Pull request merged**